modules/react/breadcrumbs/lib/hooks/useRTL.ts

```diff
@@ -5,9 +5,6 @@ import {
 } from '@workday/canvas-kit-react/common';
 
 export const useRTL = (partialTheme?: PartialEmotionCanvasTheme) => {
-  const theme = useTheme(partialTheme);
-  const shouldUseRTL = theme.canvas.direction === ContentDirection.RTL;
-  return {
-    shouldUseRTL,
-  };
+  const {canvas} = useTheme(partialTheme);
+  return {shouldUseRTL: canvas.direction === ContentDirection.RTL};
```

Comment:
If we have this hook https://github.com/Workday/canvas-kit/blob/master/modules/react/common/lib/theming/useIsRTL.ts can we remove this?